### PR TITLE
Add Quick Start Guide

### PR DESCRIPTION
**Pull request by oezkancodes**
Add Quick Start Guide to README.md explaining how to get BasicMap as fast as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+}
+```
+
+Start your app
+
+```bash
+yarn start
+```
+
 ## Getting Started
```
Comment (friedjoff) [inline README.md:82]: Can you delete the `Getting Started` part, seems duplicated to me.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
 
 Documentation and examples at https://react-spatial.geops.de.
 
+## Quick Start Guide
+
+Create a new react app
```
Comment (friedjoff) [inline README.md:15]:
```suggestion
Create a new React app:
```

File: README.md [hunk 3]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+
+```bash
+npx create-react-app react-spatial-demo
+```
+
+Navigate to your `react-spatial-demo` folder and install all dependencies
```
Comment (friedjoff) [inline README.md:21]:
```suggestion
Navigate to your `react-spatial-demo` folder and install all dependencies:
```

File: README.md [hunk 4]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+
+```bash
+yarn add ol react-spatial react-dom mapbox-gl
+```
+
+Open `App.js` and import styles, `BasicMap` and `ConfigReader`
```
Comment (friedjoff) [inline README.md:27]:
```suggestion
Open `App.js` and import styles, `BasicMap` and `ConfigReader`:
```

File: README.md [hunk 5]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+
+```js
+import 'react-spatial/themes/default/index.scss';
+import 'ol/ol.css';
+import BasicMap from 'react-spatial/components/BasicMap';
+import ConfigReader from 'react-spatial/ConfigReader';
+```
+
+Add baselayer configuration below
```
Comment (friedjoff) [inline README.md:36]:
```suggestion
Add baselayer configuration below:
```

File: README.md [hunk 6]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+  name: 'OSM Baselayer',
+  visible: true,
+  data: {
+    type: 'xyz',
+    url: 'https://c.tile.openstreetmap.org/{z}/{x}/{y}.png',
+  },
+}]);
+```
+
+Add `BasicMap` with `layers` prop to initialize the baselayer
```
Comment (friedjoff) [inline README.md:49]:
```suggestion
Add `BasicMap` with `layers` prop to initialize the baselayer:
```

File: README.md [hunk 7]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+function App() {
+  return (
+    <div className="map-wrapper">
+      <BasicMap layers={layers}/>
+    </div>
+  );
+}
+```
+
+Navigate to your `App.css` and set the wrapper to absolute and give the map full height
```
Comment (friedjoff) [inline README.md:61]:
```suggestion
Navigate to your `App.css` and set the wrapper to absolute and give the map full height:
```

File: README.md [hunk 8]
```diff
@@ -10,6 +10,75 @@ This library provides React components to build web applications based on [OpenL
...
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+.map-wrapper .rs-map {
+  height: 100%;
+}
+```
+
+Start your app
```
Comment (friedjoff) [inline README.md:76]:
```suggestion
Start your app:
```

**Comment by TroubleBeast** [comment]
I removed 'Getting Started' and formatted the requested parts as you wished

**Comment by TroubleBeast** [comment]
@mhaertwig reminder

**Comment by lb1c** [comment]
Ping @friedjoff / @edellucien

**Comment by oterral** [comment]
too old to be useful

**Pull request closed without merging**